optaplanner-wb-ui/pom.xml

```diff
@@ -1424,4 +1434,69 @@
 
   </build>
 
+  <profiles>
+    <profile>
+      <id>not-ts-exporter-build</id>
```

Comment:
Would you maybe put there a comment explaining why is this configuration needed?

Reply:
This was removed.